tests/stress/ebpf_mt_stress.h

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <winsock2.h>
+#include <windows.h>
+#include <ws2tcpip.h>
+#include <chrono>
+#include <filesystem>
+#include <format>
+#include <intsafe.h>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <ranges>
+#include <semaphore>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <thread>
+#include <utility>
+#include <variant>
+
+// Logging support
+enum class log_level : uint32_t
+{
+    LOG_ERROR = 1,
+    LOG_WARN,
+    LOG_INFO,
+    LOG_VERBOSE,
+    LOG_DEBUG
+};
+
+template <typename... Args>
+inline static void
+_log(log_level msg_level, const std::string_view& fmt, Args&&... args)
+{
+    extern log_level cur_log_level;
+    if (msg_level <= cur_log_level) {
+        std::cout << std::vformat(fmt, std::make_format_args(args...)) << "\n";
+    }
+}
+
+#define LOG_ERROR(fmt, ...) _log(log_level::LOG_ERROR, fmt, ##__VA_ARGS__)
+#define LOG_WARN(fmt, ...) _log(log_level::LOG_WARN, fmt, ##__VA_ARGS__)
+#define LOG_INFO(fmt, ...) _log(log_level::LOG_INFO, fmt, ##__VA_ARGS__)
+#define LOG_VERBOSE(fmt, ...) _log(log_level::LOG_VERBOSE, fmt, ##__VA_ARGS__)
+#define LOG_DEBUG(fmt, ...) _log(log_level::LOG_DEBUG, fmt, ##__VA_ARGS__)
+
+// Test control info.
+struct jit_test_control
+{
+    // The number of threads allocated per jit program.
+    uint32_t threads_count{0};
+
+    // The run time for each jit program test thread in seconds.
+    uint32_t duration{0};
+
+    // Flag to enable verbose progress output.
+    bool verbose_output{false};
+
+    // Flag to enable continous stop-and-restart of the extension.
+    bool extension_restart_enabled{false};
+
+    // Delay between extension restarts (in milliseconds)
```

Comment:
nit: most comments end in a period.  This one and line 71 don't.  Consistency would be good.

Reply:
fixed.